plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergFileOperations.java

```diff
@@ -899,6 +907,59 @@ public void testShowTables()
         assertFileSystemAccesses("SHOW TABLES", ImmutableMultiset.of());
     }
 
+
+    public static Stream<Arguments> partitionTransformFunctions()
+    {
+        return Stream.of(
+                Arguments.of("hour(d)", 4),
+                Arguments.of("day(d)", 2),
+                Arguments.of("month(d)", 2),
+                Arguments.of("year(d)", 2),
+                Arguments.of("bucket(d, 4)", 2),
+                Arguments.of("truncate(d, 4)", 2));
+    }
+
+    @ParameterizedTest
```

Comment:
AFAIK the use of Parameterized tests is discouraged in `trinodb/trino`.
Use `io.trino.plugin.iceberg.BaseIcebergConnectorTest#testDecimal` as reference to change the code.